src/pages/leaderboard.js

```diff
@@ -128,10 +128,11 @@ const Leaderboard = ({ data }) => {
   };
 
   const filterOptions = [
-    { value: "Last 60 days", label: "Last 60 days" },
-    { value: "Last 90 days", label: "Last 90 days" },
     { value: "2022", label: "2022" },
     { value: "2021", label: "2021" },
+    { value: "Last 60 days", label: "Last 60 days" },
+    { value: "Last 90 days", label: "Last 90 days" },
+    { value: "All time", label: "All time" },
```

Comment:
This works by accident (there's a default case in `filterFindingsByTimeFrame`, so "All time" will have the intended result of all findings)